contracts/erc721/UserMintableERC721.sol

```diff
@@ -126,9 +134,9 @@ contract UserMintableERC721 is Initializable, ERC721, ERC721Enumerable, ERC721Me
       abi.encodePacked(
         _tokenUriBase,
         "0x",
-        Utilities.uintToHexStr(uint256(_getOwnAddress())),
+        Utilities.uintToHexStrPadded(uint256(_getOwnAddress()), 20),
```

Comment:
Here we make sure that there are leading 0s if needed.

Reply:
Maybe we should add comment on why do we use the uintToHexStrPadded. It is only when you do uint(bytesn). In the tokenID I do not think it matters that much.